hudi-client/hudi-client-common/src/test/java/org/apache/hudi/client/embedded/TestEmbeddedTimelineService.java

```diff
@@ -43,7 +44,7 @@ public class TestEmbeddedTimelineService extends HoodieCommonTestHarness {
 
   @Test
   public void embeddedTimelineServiceReused() throws Exception {
-    HoodieEngineContext engineContext = new HoodieLocalEngineContext(new Configuration());
+    HoodieEngineContext engineContext = new HoodieLocalEngineContext(HadoopFSUtils.getStorageConf(new Configuration()));
```

Comment:
Can we use `HoodieTestUtils.getDefaultStorageConf()` ?

Reply:
Fixed.